buildkitd/dockerd-wrapper.sh

```diff
@@ -71,7 +76,30 @@ execute() {
     return "$exit_code"
 }
 
+find_free_subdir() {
```

Comment:
Out of curiosity, why use (and test for the existence of) 1-100 instead of using a temp name? I suppose the former limits garbage to clean up. Is that the idea?